src/containers/FilmFrontpage/FilmContentCard.tsx

```diff
@@ -28,140 +24,63 @@ interface MovieType {
-  font-weight: ${fonts.weight.semibold};
-  ${mq.range({ from: breakpoints.tablet })} {
-    ${fonts.size.text.metaText.medium};
-  }
-  color: ${colors.text.primary};
-`;
+const ImageWrapper = styled("div", {
+  base: {
+    position: "relative",
+    overflow: "hidden",
+    borderRadius: "4px",
+  },
+});
 
-const ImageWrapper = styled.div`
-  position: relative;
-  background-color: ${colors.ndlaFilm.filmColorLight};
-  border-radius: ${misc.borderRadius};
-  overflow: hidden;
-`;
+const StyledImage = styled(Image, {
+  base: {
+    aspectRatio: "16/9",
+    width: "100%",
+    objectFit: "cover",
+  },
+});
 
-const StyledImage = styled.img`
-  aspect-ratio: 16/9;
-  width: 100%;
-  object-fit: cover;
-`;
+const StyledSafeLink = styled(SafeLink, {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "xsmall",
 
-const StyledSafeLink = styled(SafeLink)`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.small};
-  color: ${colors.black};
-  box-shadow: none;
-  &:hover,
-  &:focus-within,
-  &:active {
-    [data-title] {
-      text-decoration: underline;
-    }
-    [data-content-cards] {
-      opacity: 1;
-    }
-    img {
-      opacity: 0.7;
-    }
-  }
-  &[data-type="slideshow"] {
-    width: 30vw;
-    ${mq.range({ until: breakpoints.tablet })} {
-      width: 40vw;
-    }
-    ${mq.range({ from: breakpoints.desktop })} {
-      width: 30vw;
-    }
-  }
-  &[data-type="list"] {
-    width: 40vw;
-    ${mq.range({ from: breakpoints.tablet, until: breakpoints.desktop })} {
-      width: 28vw;
-    }
-    ${mq.range({ from: breakpoints.desktop })} {
-      width: 20vw;
-    }
-    ${mq.range({ from: breakpoints.ultraWide })} {
-      width: 14vw;
-    }
-  }
-`;
-
-const StyledWrapperDiv = styled.div`
-  transition: opacity 200ms ease;
-  padding: ${spacing.xsmall} ${spacing.xsmall};
-  opacity: 0;
-  position: absolute;
-  bottom: 0px;
-  left: 0px;
-  z-index: ${stackOrder.offsetSingle};
-`;
-
-const StyledMovieTags = styled(Text)`
-  font-weight: ${fonts.weight.semibold};
-  background: ${colors.brand.greyLight};
-  padding: ${spacing.xxsmall} ${spacing.xsmall};
-  border-radius: ${misc.borderRadius};
-  color: ${colors.text.primary};
-  margin: 0px ${spacing.xxsmall} ${spacing.xxsmall} 0px;
-`;
-
-const mappedResourceTypes = movieResourceTypes.reduce<Record<string, string>>((acc, resourceType) => {
-  acc[resourceType.id] = resourceType.name;
-  return acc;
-}, {});
+    width: "40vw",
+    tabletToDesktop: {
+      width: "28vw",
+    },
+    desktop: {
+      width: "20vw",
+    },
 
-const FilmContentCard = ({
-  movie: { metaImage, resourceTypes, title, id, path },
-  hideTags = false,
-  className,
-  type = "slideshow",
-  lazy,
-  ...rest
-}: Props) => {
-  const backgroundImage = metaImage ? `${metaImage.url}?${makeSrcQueryString(600)}` : "";
-  const contentTypeId = `${type}-content-type-${id}`;
+    _hover: {
+      "& > *": {
+        textDecoration: "none",
+      },
+    },
+  },
```

Comment:
Trengs dette? SafeLink applier vel ikke noe style dersom et klassenavn er satt.

Reply:
Hvis det er for å fjerne `textDecoration` på `StyledText` tipper jeg det er en penere måte å gjøre det på.

Reply:
Det virker som det er for å fjerne underline ja.
Om du vil ha en penere måte, så må du nesten si hva det er for jeg tror ikke jeg har noen mening om hva som er pent her.

Reply:
La oss bruke et data-attributt!